### ISSUE TITLE
{Linked:Bug248147}[Adaptive Card- Designer] - While navigating on the group of checkboxes or radio buttons, Voice over is conveying label information with all radio buttons.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#483]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.24.11.3.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 12
iOS Version: 26.
Application: Adaptive Cards Visualizer
Screen Reader: Voice Over

**Repro Steps:**
1. Launch the application.
2.  Perform right swipe and navigate to the version 1.5 and double tap to activate it.
3. Perform right swipe and navigate to the scenario button and double tap to activate it.
4. Perform right swipe and navigate to the 'Inputwithvalidation.JSON' option and double tap to activate it.
5. Navigate to radio and checkbox groups present under 'Input.ChoiceSet' and observe the issue.

**Actual Result:**
While navigating on the radio button option present under **'What color do you want? (Expanded)'** and checkboxes present under **'What color do you want? (Expanded)'** label, voice over is announcing label information with all radio buttons and checkboxes.
For every radio button it is announcing as ''What color do you want? expanded  radio button label.''
For every Checkboxes, it is announcing as 'What color do you want? expanded checkbox label.'

**Expected Result:**
The group label should be announced once when the user enters the group, not repeated for every checkbox or radio button.
Each item should announce only its own label when focused.

**User Impact:**
It will impact the screen reader user as they will get confused with the extra verbose and The repeated announcement of “required” with every checkbox or radio option is confusing for users. It may mislead them into thinking that each individual option must be selected,

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html?utm_source=chatgpt.com

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/de2b66cd-3f13-4232-bdf8-4781b8fe1e0f

### Card JSON

```JSON
NA
```
